internal/broker/testdata/TestIsAuthenticated/golden/error_when_mode_is_newpassword_and_fetching_user_info_fails/first_call

```diff
@@ -0,0 +1,3 @@
+access: denied
+data: '{"message":"authentication failure: could not get required information"}'
+err: <nil>
```

Comment:
As far as I'm aware, `newpassword` does not care about the userInfo, so this test is a bit confusing for me. Maybe I understood it wrong?